go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/bradleyfalzon/ghinstallation/v2
 
-go 1.22.0
+go 1.23.0
```

Comment:
This upgrade needs to happen because https://github.com/google/go-github/pull/3512 bumped to 1.23.0